babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I need to rename `babel.config.js` to `babel.config.cjs` to make it work. Otherwise I get error `error while loading config - You appear to be using a native ECMAScript module configuration file, which is only supported when running Babel asynchronously.`

Reply:
this works for me either